Reference/Templating/Mvc/partial-views.md

```diff
@@ -95,6 +95,26 @@ The above will cache the output of your partial view for one hour (3600 seconds)
 		int cachedSeconds,
 		bool cacheByPage = false,
 		bool cacheByMember = false,
-		ViewDataDictionary viewData = null)
+		ViewDataDictionary viewData = null
```

Comment:
This is missing a comma after = null 

eg it should be

ViewDataDictionary viewData = null,
Func<object, ViewDataDictionary, string> contextualKeyBuilder = null)